climada/entity/exposures/test/test_base.py

```diff
@@ -89,6 +90,20 @@ def test__init__meta_type(self):
         self.assertEqual("meta must be a dictionary",
                       str(cm.exception))
 
+    def test__init__geometry_type(self):
+        """ Check if the column name of geometry of GDF was customized"""
+        with self.assertRaises(ValueError) as cm:
+            Exposures(geometry = 'myname')
+        self.assertEqual("Exposures is not able to handle customized 'geometry' column names.",
+                      str(cm.exception))
+
+    def test__init__mda_in_kwargs(self):
+        """ Check if mda is not present in Kwargs after being removed"""
```

Comment:
```suggestion
        """Check if mda is not present in kwargs after being removed"""
```
That descriptions is a bit hard to understand.